src/components/ComponentPreview.tsx

```diff
@@ -38,31 +33,12 @@ const ComponentPreviewWrapper: React.FC<ComponentPreviewWrapperProps> = ({
         />
       </div>
       <div className="card-body">
-        {previewMode === 'editJSON' ? (
-          <JSONEdit
-            data={component}
-            rows={10}
-            aria-label={intl.formatMessage({
-              description: 'Accessible label for builder preview JSON edit field',
-              defaultMessage: 'Edit component JSON',
-            })}
-          />
+        {previewMode === 'JSON' ? (
+          <JSONEditor height="45vh" value={component} onChange={setValues} />
```

Comment:
can't you set max-block-size instead of height? The former is writing mode-agnostic.